common/app/views/fragments/items/facia_cards/contentCard.scala.html

```diff
@@ -88,10 +88,10 @@
                 </div>
             }
 
-            case Some(media@InlineYouTubeMediaAtom(_)) => {
+            case Some(media@InlineYouTubeMediaAtom(_)) if item.cardTypes.showYouTubeMediaAtomPlayer => {
```

Comment:
Let's see how much the template varies from the current playable one. It may be cleaner to make it its own fragment like `posterOnly` case.